.github/lineage.yml

```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```

Comment:
This should be a descendant of `skeleton-tf-module`.

Reply:
See commit 8dddb16.